firewalls.yml

```
@@ -80,15 +80,15 @@
     ansible_become: no
   roles:
     # ...
-    - role: opnsense-facts
+    - role: ansible-opnsense-facts
       tags:
         - opnsense
         - facts
         - always
-    - role: opnsense # https://github.com/Rosa-Luxemburgstiftung-Berlin/ansible-opnsense
+    - role: ansible-opnsense # https://github.com/Rosa-Luxemburgstiftung-Berlin/ansible-opnsense
       tags:
         - opnsense
```

ok, in order to make it a complete example, we have to add some more roles her ...
```suggestion
        - opnsense
    - role: flushhandlers  # https://github.com/zerwes/ansible-role-flushhandlers
      tags: always
    - role: ansible-opnsense-update  # https://github.com/Rosa-Luxemburgstiftung-Berlin/ansible-opnsense-update
      tags:
        - opnsense
        - update
    - role: ansible-opnsense-plugpack  # https://github.com/Rosa-Luxemburgstiftung-Berlin/ansible-opnsense-plugpack
      tags:
        - opnsense
        - plugpack
        - plugins
        - packages
```